easybuild/easyconfigs/b/BlobTools/BlobTools-20180528-foss-2018b-Python-2.7.15.eb

```diff
@@ -13,21 +14,24 @@ toolchain = {'name': 'foss', 'version': '2018b'}
 
 source_urls = [
     'https://github.com/drl/blobtools/archive/',
-    'ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/',
+    'ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/taxdump_archive',
 ]
 sources = [
     {
         'download_filename': '%s.tar.gz' % commit,
         'filename': SOURCE_TAR_GZ,
         'extract_cmd': "tar -xzf %s -C %(builddir)s --strip-components 1",
     },
-    {'filename': 'taxdump.tar.gz', 'extract_cmd': "cp %s %(builddir)s/data/"},
+    {
+        'filename': 'taxdmp_%s.zip' % taxdump_ver,
+        'extract_cmd': 'unzip %s && cp nodes.dmp names.dmp %(builddir)s/data/',
```

Comment:
@smoors Maybe use `cp -a *  %(builddir)s/data/`?

Reply:
@boegel only nodes.dmp names.dmp are needed, and this is also what the installation script does.
I agree it's better to use `cp -a` though.

Reply:
OK, just `cp -a` then :)